src/features/account/screens/register.screen.js

```diff
@@ -56,13 +59,17 @@ export const RegisterScreen = ({ navigation }) => {
           </ErrorContainer>
         )}
         <Spacer size="large">
-          <AuthButton
-            icon="email"
-            mode="contained"
-            onPress={() => onRegister(email, password, repeatedPassword)}
-          >
-            Register
-          </AuthButton>
+          {!isLoading ? (
```

Comment:
Don't forget to add ```setIsLoading(true``` to the beginning of the `onRegister` function in the context provider!

Reply:
Got it